index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/43fc886676.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Gaming Convention</title>
+  </head>
+  <body>
+    <div id="container">
+      <header class="head-section">
+        <div class="hamburger-icon">
+          <button class="hamburger-button" type="button">
+            <span class="h-button"></span>
+            <span class="h-button"></span>
+            <span class="h-button"></span>
+          </button>
+        </div>
+        <div class="nav-bars">
+          <nav class="nav-bar-one">
+            <ul>
+              <li>
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+              </li>
+              <li><a href="#">English</a></li>
+              <li><a href="#">My Page</a></li>
+              <li><a href="#">Logout</a></li>
+            </ul>
+          </nav>
+          <nav class="nav-bar-two">
+            <div class="">
+              <ul>
+                <li><a href="index.html">Gaming convention 2022</a></li>
+              </ul>
+            </div>
+            <div class="navs-two">
+              <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+              </ul>
+            </div>
+            <div class="campaign">
+              <ul>
+                <li><a href="#">CC Campaign</a></li>
+              </ul>
+            </div>
+          </nav>
+        </div>
+      </header>
+      <main>
+        <section id="header-content">
+          <h1>Welcome All!</h1>
+          <div class="bar-game">
+            <h2>Barbados Gaming Convention 2022</h2>
+          </div>
+          <div class="paragraph">
+            <p class="paragraph-one">
+              Come and enjoy some of the world's best gaming consoles and
+              computer gaming desktops built by our very own, enjoy the light
+              show and play the latest games. Compititions will also be held
+              with amazing prizes.
+            </p>
+          </div>
+          <div class="date">
+            For Two days, May-14th-2022 until (Sun)May-15th-2022
+          </div>
+          <p class="sentence-one">@ Full Scale Gaming Auditorium</p>
+        </section>
+        <section id="program">
+          <h3>Main Program</h3>
+          <div class="program-block">
+            <div class="gaming one">
```

Comment:
- [ ] For the mobile version of the index page, I would advise giving the main program container width of 100% and some margin to each individual card so as to get rid of that white space around the main container for the cards.

Your design
![image](https://user-images.githubusercontent.com/81932478/169436446-33b43971-6486-434f-8116-eca8d04f6a42.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/169436494-da3bb450-810e-44ef-8bd7-e9706e630158.png)

Reply:
fixed

Reply:
Implemented 👍